src/screens/WasteCollectionScreen.js

```diff
@@ -36,42 +45,12 @@ import { getInAppPermission, showPermissionRequiredAlert } from '../pushNotifica
 import { getQuery, QUERY_TYPES } from '../queries';
 import { SettingsContext } from '../SettingsProvider';
 
-const dotSize = 6;
-
 setupLocales();
 
-const getMarkedDates = (types, streetData) => {
-  let markedDates = {};
-
-  const wasteLocationTypes = streetData?.[QUERY_TYPES.WASTE_ADDRESSES]?.[0]?.wasteLocationTypes;
+const { CALENDAR } = consts;
+const { DOT_SIZE } = CALENDAR;
 
-  if (wasteLocationTypes && types) {
-    wasteLocationTypes.forEach((wasteLocationType) => {
-      // only add marked dates for known types
-      if (!types[wasteLocationType.wasteType]) {
-        return;
-      }
-
-      const { color, selected_color: selectedColor } = types[wasteLocationType.wasteType];
-      wasteLocationType?.listPickUpDates?.forEach((date) => {
-        markedDates[date] = {
-          dots: [...(markedDates[date]?.dots ?? []), { color, selectedColor }]
-        };
-      });
-    });
-  }
-
-  const today = moment().format('YYYY-MM-DD');
-
-  // highlight today
-  markedDates[today] = {
-    ...(markedDates[today] ?? {}),
-    selected: true,
-    selectedColor: colors.lighterPrimary
-  };
-
-  return markedDates;
-};
+const today = moment().format('YYYY-MM-DD');
```

Comment:
We use the same code in many places. Why don't we create a helper for that?

Reply:
haven't found the time to think about it and find a good refactoring.